tests/dragonfly/replication_test.py

```diff
@@ -2626,3 +2626,34 @@ async def test_replica_of_replica(df_factory):
         await c_replica2.execute_command(f"REPLICAOF localhost {replica.port}")
 
     assert await c_replica2.execute_command(f"REPLICAOF localhost {master.port}") == "OK"
+
+
+@pytest.mark.asyncio
+@pytest.mark.slow
+async def test_big_containers(df_factory):
```

Comment:
I believe that this test should be one test case for test_replication_all in which you play different config params